### ISSUE TITLE
Blank page on checkout window

### ISSUE THREAD

**Issue by OrkhanKerimli** [PaddleHQ/Mac-Framework-V4#40]
Hi, 

When I open checkout dialog with a "showCheckout" method, it opens a new window with a blank page.
Checkout page is completely white - content is not visible. 
If move mouse on window, it is visible that all content loaded - cursor changes for buttons and text fields. But nothing is visible.
[Screenshot](http://prntscr.com/lwdk3a)

Debug mode shows nothing and I do not get any error or warning message.
My macOS version is 10.12.6.

Code: 

        // Your Paddle SDK Config from the Vendor Dashboard:
        let myPaddleVendorID = "123"
        let myPaddleAPIKey = "123abc"
        let myPaddleProductID = "789"

        // Default Product Config in case we're unable to reach our servers on first run:
        let defaultProductConfig = PADProductConfiguration()
        defaultProductConfig.productName = "My v4 Product"
        defaultProductConfig.vendorName = "My Company"

        // Initialize the SDK singleton with the config:
        let paddle = Paddle.sharedInstance(withVendorID: myPaddleVendorID, apiKey: myPaddleAPIKey, productID: myPaddleProductID, configuration: defaultProductConfig)
        Paddle.enableDebug()

        // Initialize the Product you'd like to work with:
        let paddleProduct = PADProduct(productID: myPaddleProductID, productType: PADProductType.sdkProduct, configuration: defaultProductConfig)
        paddle?.delegate = self

        // Ask the Product to get it's latest state and info from the Paddle Platform:
        paddleProduct?.refresh({ (delta: [AnyHashable : Any]?, error: Error?) in
                        
            paddle?.showCheckout(for: paddleProduct!, options: nil, checkoutStatusCompletion: {{ (state: PADCheckoutState?, data: PADCheckoutData?) in
                // completion handler
                }}())
        })

**Comment by GregularExpressions** [comment]
Hey @OrkhanKerimli 

Thanks for the report.

Two initial things to check please:

1. Is `webkit` included in the project? (Link Binary with Libraries in build phases).
2. Are you able to load a test checkout directly in Safari? `[https://pay.paddle.com/checkout/544175](https://pay.paddle.com/checkout/544175)`

Thanks,
Greg.

**Comment by OrkhanKerimli** [comment]
@GregularExpressions, Yes, WebKit framework included and test checkout loads in Safari without any problem.

**Comment by GregularExpressions** [comment]
@OrkhanKerimli thanks for the quick response, is it possible to get a build that exhibits this behaviour please? You can email me; greg@paddle.com.

**Comment by OrkhanKerimli** [comment]
Dear @GregularExpressions, I send an email. Please, check it.

**Comment by firede** [comment]
Hi, @GregularExpressions 
I got the same issue, is there any progress? Thanks.

**Comment by firede** [comment]
> Hi, @GregularExpressions
> I got the same issue, is there any progress? Thanks.

Issue solved (because the product is not released), thanks.

**Comment by firede** [comment]
@OrkhanKerimli 
Look at your product list: https://vendors.paddle.com/products

If you can see `Product State` column, `Update` it, until this item is Green:
* Before:
![image](https://user-images.githubusercontent.com/157338/50455289-01afa000-0988-11e9-8a04-c4a67ae00646.png)
* After:
![image](https://user-images.githubusercontent.com/157338/50455333-5226fd80-0988-11e9-8afe-411896de90e1.png)


Finally, your product list will like this:
![image](https://user-images.githubusercontent.com/157338/50455314-24da4f80-0988-11e9-93d1-90d0168d3019.png)

**Comment by OrkhanKerimli** [comment]
@firede, thank you for detailed answer, but my product already has been released and unfortunately, your solution is not working for me.
